newIDE/app/src/Export/PreviewLauncher.flow.js

```diff
@@ -75,3 +57,20 @@ export type HotReloaderLog = {|
   kind: 'fatal' | 'error' | 'warning' | 'info',
   message: string,
 |};
+
+/** The functions that PreviewLauncher must expose on their class */
+export type PreviewLauncherInterface = {
+  launchPreview: (previewOptions: PreviewOptions) => Promise<any>,
+  canDoNetworkPreview: () => boolean,
+  canDoHotReload: () => boolean,
+  +getPreviewDebuggerServer: () => ?PreviewDebuggerServer,
```

Comment:
I just realised this function has a + at the beginning. Is that on purpose?

Reply:
Yes I think. It's to comply with Flow exact types